src/components/button/button.ts

```diff
@@ -98,6 +98,9 @@ export default class SlButton extends ShoelaceElement implements ShoelaceFormCon
   /** An optional value for the button. Ignored when `href` is set. */
   @property() value = '';
 
+  /** The button's title attribute. */
```

Comment:
This isn't obvious, but the docs will ignore properties that don't have JSDoc comments like `/**`. We probably _don't_ want to document this, since it's a global attribute. Instead, let's move these up closer to the private properties and change the comment so we know _why_ we're adding them, e.g.

```ts
@property() title = ''; // make reactive to pass through
```

Reply:
Alright. Done.